src/java.base/share/classes/java/lang/classfile/package-info.java

```diff
@@ -256,17 +256,31 @@
  * unresolved labels) affects internal tools and may cause exceptions later in
  * the classfile building process.
  * <p>
- * Basic syntax control can be achieved using symbolic descriptors. For example
- * using {@link java.lang.constant.ClassDesc} assures the right serial form is
- * used in the particular place inside classfile and also assures the class name
- * has been already syntactically validated.
+ * Syntax validation is applied during symbolic descriptors construction.
+ * For example {@link java.lang.constant.ClassDesc#of} validates the class binary
+ * name or {@link java.lang.constant.ClassDesc#ofDescriptor} validates the class
+ * descriptor syntax.
+ * <p>
+ * Using symbolic descriptors assures the right serial form selection by
+ * the ClassFile API library. For example a class name is stored in its internal
```

Comment:
I find this comment still very hard to parse. There is a lot of terminology here thrown to the reader (internal form, class descriptor, serial form), and very little explanation of which is which. I get the general idea that, if I use a `ClassDesc`, the API will take care to convert that to the right form, depending on the context - but it feels there's too much text, and too few examples to ground that text.

Reply:
I've tried to rephrase the paragraph and simplify it, thanks.